Xamarin.Forms.Platform.Android/Renderers/ListViewAdapter.cs

```diff
@@ -309,6 +311,68 @@ public override AView GetView(int position, AView convertView, ViewGroup parent)
 			return layout;
 		}
 
+		internal void InvalidatePrototypicalCellCache()
+		{
+			_prototypicalCellByTypeOrDataTemplate.Clear();
```

Comment:
TODO: Invalidate the cache if the `DataTemplateSelector` changes. For now, just wanna get this in while I have a CI baseline.